components/breadcrumb/stories/breadcrumb.stories.js

```diff
@@ -48,13 +61,252 @@ Default.args = {
 		},
 		{
 			label: "Trend",
+		},
+		{
+			label: "January 2019 Assets",
+		},
+	],
+};
+
+export const MultilineNested = Template.bind({});
+MultilineNested.args = {
+	items: [
+		{
+			iconName: "FolderOpen",
+		},
+		{
+			label: "Sub Item",
```

Comment:
Minor nit to try to keep headings to sentence case per our content [style guide](https://spectrum.adobe.com/page/grammar-and-mechanics/#Capitalization). I see a few other minor changes needed below for this too.

```suggestion
			label: "Sub item",
```